mungegithub/mungers/lgtm_handler.go

```diff
@@ -83,7 +98,46 @@ func (LGTMAfterCommitMunger) Munge(obj *github.MungeObject) {
 	}
 }
 
-func (LGTMAfterCommitMunger) isStaleComment(obj *github.MungeObject, comment *githubapi.IssueComment) bool {
+func hasValidReviwer(reviewers []*githubapi.User) bool {
+	for _, r := range reviewers {
+		if r != nil && r.Login != nil {
+			return true
+		}
+	}
+	return false
+}
+
+func foundLGTMFromReviewer(comments []*githubapi.IssueComment, reviewers []*githubapi.User, lastModified *time.Time) bool {
+	for _, c := range comments {
+		if lastModified == nil || c.CreatedAt == nil || (*lastModified).After(*c.CreatedAt) {
+			continue
+		}
+		if !isReviewer(c.User, reviewers) {
```

Comment:
@hongchaodeng Yes, I just don't want someone to think they put lgtm and then wonder why the label didn't get added.

Also note that PRs can be manually assigned, and might be assigned to someone not in the OWNERS file. Such reviews can still be helpful, but someone in the OWNERS file should still be required to sign off.